examples/tutorials/research/02_advanced_usage_of_research.ipynb

```diff
@@ -0,0 +1,1023 @@
+{
```

Comment:
<p>Move <code>run_later</code> into the pipeline chain.</p><p>It look a bit strange sitting outside</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>